### PR TITLE
Multiclass label support

### PR DESCRIPTION
**Pull request by brandon-groundlight**
Tests and labeling support for multiclass detectors

### CODE REVIEW

File: src/groundlight/binary_labels.py [hunk 1]
```diff
@@ -53,25 +53,3 @@ def convert_internal_label_to_display(
 
     logger.warning(f"Unrecognized internal label {label} - leaving it alone as a string.")
     return label
-
-
-def convert_display_label_to_internal(
```
Comment (tyler-romero) [inline src/groundlight/binary_labels.py:58]: 🥳

File: src/groundlight/client.py [hunk 2]
```diff
@@ -1066,16 +1068,20 @@ def _wait_for_result(
...
+        rois: Union[List[ROI], str, None] = None,
     ):
         """
         Provide a new label (annotation) for an image query. This is used to provide ground-truth labels
         for training detectors, or to correct the results of detectors.
 
         **Example usage**::
 
-            gl = Groundlight()
+            gl = ExperimentalApi()
```
Comment (tyler-romero) [inline src/groundlight/client.py:1084]: Why does this need to be `ExperimentalAPI()`

**Review by tyler-romero — APPROVED** [review]
LGTM! Thanks for adding those new tests too

File: src/groundlight/client.py (hunk 2 above)
Comment (brandon-groundlight) [reply src/groundlight/client.py:1084]: Copy paste, my greatest power and greatest weakness

**Comment by tyler-romero** [comment]
I'm seeing backend errors that I believe are related to this PR. Specifically, submitting counting labels triggers count-detector training. But our create_count_detector sdk method doesnt allow specifying what the object to be counted is. This bad specification of count detectors causes errors during training.

File: test/unit/test_labels.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    gl_experimental.add_label(iq1, "apple")
+    iq1 = gl_experimental.get_image_query(iq1.id)
+    assert iq1.result.label == "apple"
+    gl_experimental.add_label(iq1, "banana")
+    iq1 = gl_experimental.get_image_query(iq1.id)
+    assert iq1.result.label == "banana"
+    gl_experimental.add_label(iq1, "cherry")
+    iq1 = gl_experimental.get_image_query(iq1.id)
+    assert iq1.result.label == "cherry"
+    # You can submit the index of the class as well
```
Comment (srnangi) [inline test/unit/test_labels.py:53]: Ohh, I see that we can use class index in labels. Should we add this to function description for `add_label` too?

**Review by srnangi — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**